.test-infra/metrics/grafana/dashboards/perftests_metrics/Python_IO_IT_Tests_Dataflow.json

```diff
@@ -35,7 +35,7 @@
       },
       "hiddenSeries": false,
       "id": 2,
-      "interval": "",
+      "interval": "15h",
```

Comment:
Is there any particular reason why "15"?

Reply:
I chose 15h because this interval is set to the AutoJob in jenkins. The java FileBased dashboards have it set to this value (in this case 6h)

Reply:
"H 15 * * *" means the job is executed daily at 15:00 (3 p.m.) UTC. So the interval should be 24h

Reply:
Take a look at the job history: https://builds.apache.org/view/A-D/view/Beam/view/All/job/beam_PubsubIO_Performance_Test_Python/

Reply:
I didn't dig into it, bu now it's clear. I changed to 24h